bluesky_kafka/tests/conftest.py

```diff
@@ -1,7 +1,23 @@
+import intake
+import msgpack
+import msgpack_numpy as mpn
+import numpy as np
+import ophyd.sim
+import os
 import pytest
+import tempfile
+import yaml
 
 from bluesky.tests.conftest import RE
-import ophyd.sim
+from bluesky_kafka import Publisher
+from functools import partial
+
+
+TEST_TOPIC = "bluesky-kafka-test"
+TEST_TOPIC2 = "bluesky2-kafka-test"
+TMP_DIR = tempfile.mkdtemp()
```

Comment:
I think there should be a fixture for the intake catalog file.

Reply:
Does the way that I changed it around seem okay?